rslearn/train/prediction_writer.py

```diff
@@ -154,7 +160,8 @@ def write_on_batch_end(
 
             # This is the last patch so it's time to merge outputs from overlapped patches
             if self.merger is not None:
-                pending_output = self.merger.merge(pending_output)
+                # TODO: HackyNot sure why we don't pass metadata to the merger can't see any subclasses
```

Comment:
I updated the PatchPredictionMerger to just take output since that matches rslp.utils.nms.NMSDistanceMerger.